core/chains/evm/txmgr/helpers_test.go

```diff
@@ -1,15 +1,44 @@
 package txmgr
 
-import evmclient "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
+import (
+	"context"
 
-func SetEthClientOnEthConfirmer(ethClient evmclient.Client, ethConfirmer *EthConfirmer) {
+	"github.com/smartcontractkit/chainlink/v2/common/types"
+	evmclient "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
+)
+
+func processUnstartedEthTxsNoOp[ADDR types.Hashable[ADDR]](ctx context.Context, fromAddress ADDR) (retryable bool, err error) {
+	return false, nil
+}
+
+func SetEthClientOnEthConfirmer(ethClient evmclient.Client, ethConfirmer *EvmConfirmer) {
 	ethConfirmer.ethClient = ethClient
 }
 
-func SetResumeCallbackOnEthBroadcaster(resumeCallback ResumeCallback, ethBroadcaster *EthBroadcaster) {
+func SetResumeCallbackOnEthBroadcaster(resumeCallback ResumeCallback, ethBroadcaster *EvmBroadcaster) {
 	ethBroadcaster.resumeCallback = resumeCallback
 }
 
-func (er *EthResender) ResendUnconfirmed() error {
+func (eb *EthBroadcaster[ADDR, TX_HASH, BLOCK_HASH]) StartInternal() error {
+	return eb.startInternal()
+}
+
+func (eb *EthBroadcaster[ADDR, TX_HASH, BLOCK_HASH]) CloseInternal() error {
+	return eb.closeInternal()
+}
+
+func (eb *EthBroadcaster[ADDR, TX_HASH, BLOCK_HASH]) DisableUnstartedEthTxAutoProcessing() {
+	eb.processUnstartedEthTxsImpl = processUnstartedEthTxsNoOp[ADDR]
```

Comment:
nit/ you might be able to avoid some generics if you use an anonymous func instead of declaring `processUnstartedEthTxsNoOp`:
```suggestion
	eb.processUnstartedEthTxsImpl = func(ctx context.Context, fromAddress ADDR) (retryable bool, err error) { return false, nil }
```